src/ITR/data/base_providers.py

```diff
@@ -1,55 +1,48 @@
-import ITR
```

Comment:
Ditto here.  We can delay importing ITR after things like `os` and `warnings`, but always must be before `pint`.